fw/http.c

```diff
@@ -6883,6 +6884,21 @@ static TfwConnHooks http_conn_hooks = {
 	.conn_send	= tfw_http_conn_send,
 };
 
+static int
+tfw_http_start(void)
+{
+	TfwVhost *dflt_vh = tfw_vhost_lookup_default();
+
+	if (tfw_blk_flags & TFW_BLK_ATT_REPLY && dflt_vh
+	    && dflt_vh->frang_gconf->ip_block)
+		T_WARN_NL("Directive 'block action' can't be set to\n"
+			  "    'attack reply' if 'ip_block' from 'frang_limits' group is 'on'\n"
+			  "    (this is misconfiguration, see the wiki).\n");
+	tfw_vhost_put(dflt_vh);
+
+	return 0;
```

Comment:
We should return -1 in case when configuration is bad i think.

Reply:
Fixed, also I'll fix all tests where this misconfiguration is used.